src/gui/mainwindow.cpp

```diff
@@ -27,6 +27,10 @@
 
 namespace gpui {
 
+#define MAIN_WINDOW_GEOMETRY "mainwindow/geometry"
+#define MAIN_WINDOW_STATE "mainwindow/state"
+#define MAIN_WINDOW_SPLITTER_STATE "mainwindow/splitterState"
+
```

Comment:
- [ ] Please use const QString instead of defines.
- [ ] Please provide conventional commit message to this commit: https://www.conventionalcommits.org/en/v1.0.0/ e.g. "feat: add ability to save application geometry state"